Article/Article.js

```diff
@@ -3,24 +3,36 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.domElement.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.textContent = "expand";
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.isOpen = false;
+    this.expandButton.addEventListener('click', () => this.expandArticle(event));
```

Comment:
You function doesn't take or use `event`, so why pass it as an argument?
You also ought to pass it as a parameter if you're going to use it:
```javascript
this.expandButton.addEventListener('click', ev => this.expandArticle(ev)); 
```
otherwise most browsers won't be running your `expandButton` listener callback because it'll just error out (since `event` isn't defined, but Chrome just happens to handle that for you).